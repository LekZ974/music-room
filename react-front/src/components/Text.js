import React from 'react';

import { Text as NBText } from 'native-base';

const Text = props => {
  return <NBText {...props} />;
};
export default Text;
