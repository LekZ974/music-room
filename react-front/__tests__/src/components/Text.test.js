import React from 'react';
import renderer from 'react-test-renderer';
import Text from '../../../src/components/Text';

it('renders correctly', () => {
  const tree = renderer.create(<Text>Snapshot test with Text Component!</Text>).toJSON();

  expect(tree).toMatchSnapshot();
});
