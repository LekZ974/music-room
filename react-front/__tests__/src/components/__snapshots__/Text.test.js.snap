// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#000",
      "fontFamily": "System",
      "fontSize": 16,
    }
  }
  uppercase={false}
>
  Snapshot test with Text Component!
</Text>
`;
